docs/elements.md

```diff
@@ -1747,6 +1748,37 @@ It expects results to follow [the reference schema for external grading results]
 
 - [External Grading Reference Schema](externalGrading.md#grading-result)
 
+
+-----
+
+
+### `pl-feedback` element
```

Comment:
What's our thinking about manual grading feedback from in-browser manual grading? I feel like it would be best if instructors don't have to put a `<pl-feedback>` element into their manually-graded questions, but instead we just display the feedback automatically.

Are we thinking that it's still helpful to have this element for grade upload feedback?

Reply:
You're probably right, but for grade upload you still need something. I'm ok with closing this altogether and discussing this in the context of manual grading, since that may be more long-term plan. I have this element in my courses anyway, in the meantime.

Reply:
Yeah, I'm thinking that we probably want to handle manual feedback (at least from in-browser grading) outside of the regular element flow. Let's close this for now and think harder about manual grading.